controllers/assignments.py

```diff
@@ -660,8 +695,35 @@ def _autograde_one_q(course_name, sid, question_name, points, question_type, dea
     if save_score:
         _save_question_grade(sid, course_name, question_name, score, id)
 
+    if practice_start_time:
+        page_visits = db((db.useinfo.course_id == course_name) & \
+                         (db.useinfo.sid == sid) & \
+                         (db.useinfo.event == 'page') & \
+                         (db.useinfo.timestamp >= practice_start_time)) \
+            .select()
+        practice_duration = (datetime.datetime.now() - practice_start_time).seconds / 60
+        practice_score = 0
```

Comment:
Please remove the print statements or comment them out.

Reply:
In my last commit, I had already removed all the print statements. There are only two of them remaining that are yours.